app/code/core/Mage/Adminhtml/controllers/Customer/AttributeController.php

```diff
@@ -0,0 +1,328 @@
+<?php
+/**
+ * Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magento.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Magento to newer
+ * versions in the future. If you wish to customize Magento for your
+ * needs please refer to http://www.magento.com for more information.
+ *
+ * @category    Mage
+ * @package     Mage_Adminhtml
+ * @copyright  Copyright (c) 2006-2020 Magento, Inc. (http://www.magento.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+class Mage_Adminhtml_Customer_AttributeController extends Mage_Adminhtml_Controller_Action
+{
+    protected $_entityTypeId;
+
+    public function preDispatch()
+    {
+        $this->_setForcedFormKeyActions('delete');
+        parent::preDispatch();
+        $this->_entityTypeId = Mage::getModel('eav/entity')->setType(Mage_Customer_Model_Customer::ENTITY)->getTypeId();
+        return $this;
+    }
+
+    protected function _initAction()
+    {
+        $this
+            ->_title($this->__('Customer'))
+            ->_title($this->__('Attributes'))
+            ->_title($this->__('Manage Customer Attributes'));
+
+        if($this->getRequest()->getParam('popup')) {
+            $this->loadLayout('popup');
+        } else {
+            $this->loadLayout()
+                ->_setActiveMenu('customer/attributes')
+                ->_addBreadcrumb(Mage::helper('customer')->__('Customer'), Mage::helper('customer')->__('Customer'))
+                ->_addBreadcrumb(
+                    Mage::helper('customer')->__('Manage Customer Attributes'),
```

Comment:
The newly added lines do not have a translation.

Reply:
fixed!